### PR TITLE
update delegation program to remove info about the 240 addresses

### PR DESCRIPTION
**Pull request by iregina**
MF has made some updates / want to reverse the plans regarding the 240 addresses

### CODE REVIEW

File: docs/node-operators/foundation-delegation-program.mdx [hunk 1]
```diff
@@ -136,19 +130,12 @@ See the following uptime leaderboards for the latest uptime performance scores:
 
 #### Payout Addresses
 
-You must send computed rewards as described below to a Mina Foundation address - however, beginning in Cycle 9, Mina Foundation changed the method of returning rewards to the Foundation.
+You must send computed rewards as described below to a Mina Foundation address.
```
Comment (garethtdavies) [inline docs/node-operators/foundation-delegation-program.mdx:133]: If just sending it back to the same address that was delegated how does this differ from O(1) and Foundation? If it is indeed the same this line seems not needed and adds potential confusion?

Comment (garethtdavies) [reply docs/node-operators/foundation-delegation-program.mdx:133]: Oh ignore that, I see this whole page is only Foundation specific :)

File: docs/node-operators/foundation-delegation-program.mdx [hunk 2]
```diff
@@ -136,19 +130,12 @@ See the following uptime leaderboards for the latest uptime performance scores:
...
 
-Previously, Block Producers were asked to send rewards to the same address that delegated to them, however this is no longer the case.  In the new system, once a Block Producer is selected as one of the 240 to participate in the Delegation Program for an upcoming cycle, they will receive an email listing a specific wallet to send their rewards to.
+Block Producers must send rewards to the same address that delegated to them. Each Block Producer will receive an individual email containing one wallet address, so all payments to that address throughout the cycle can be tracked specifically to that Block Producer.  The full list of all Block Producers and their corresponding return wallets will also be publicly available soon on Mina Foundation’s Github page.
```
Comment (garethtdavies) [inline docs/node-operators/foundation-delegation-program.mdx:135]: Worth adding that you can also see this data on-chain?

**Comment by barriebyron** [comment]
the MF approved delegation doc is live 
work done in https://github.com/o1-labs/docs2/pull/400 

@iregina I am closing this issue since the addresses were removed with your earlier PR and the updated doc is approved by MF

**Pull request closed without merging**